spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue500Test.java

```diff
@@ -34,9 +31,7 @@ class Issue500Test extends AbstractIntegrationTest {
 
     @Test
     void testIssue() {
-        performAnalysis("lambdas/Issue500.class");
-        BugInstanceMatcher bugMatcher = new BugInstanceMatcherBuilder().build();
-        assertThat(getBugCollection(), containsExactly(0, bugMatcher));
+        Assertions.assertDoesNotThrow(() -> performAnalysis("lambdas/Issue500.class"));
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/500) is about the analysis throwing exception.